docs/src/components/Logo.css

```diff
@@ -0,0 +1,19 @@
+@layer components {
```

Comment:
So what's wrong with CSS modules after all? Are we going to have plain CSS, CSS modules, and Tailwind at the same time, or do you want to get rid of modules completely?

Reply:
My personal preference would be just plain CSS + Tailwind, and get rid of the current CSS Modules usage as the docs are built out.

I would have went for CSS Modules in a larger team with developers of varying skill levels, but here I don't think there's any significant problem that the tech solves for us. (I always prefix classes with component name so clashes won't happen with basic hygiene).

Do you feel OK about this or would you strongly want to use CSS Modules anyway?

Reply:
I would use CSS modules and not go with Tailwind at all, but it's more of a preference than an objective advantage of one tech over the other. Since you'll mostly work on the docs, it's your call.